tests/greenheart/test_hydrogen/test_greenheart_system.py

```diff
@@ -95,175 +97,152 @@ def test_simulation_wind_wave(subtests):
         orbit_library_path, f"plant/hopp_config_wind_wave_solar.yaml"
     )
 
-    lcoe, lcoh, _, hi = run_simulation(
-        filename_hopp_config_wind_wave_solar,
-        filename_greenheart_config,
-        filename_turbine_config,
-        filename_orbit_config,
-        filename_floris_config,
+    config = GreenHeartSimulationConfig(
+        filename_hopp_config=filename_hopp_config_wind_wave_solar,
+        filename_greenheart_config=filename_greenheart_config,
+        filename_turbine_config=filename_turbine_config,
+        filename_orbit_config=filename_orbit_config,
+        filename_floris_config=filename_floris_config,
         verbose=False,
         show_plots=False,
         save_plots=False,
         use_profast=True,
-        post_processing=False,
+        post_processing=True,
         incentive_option=1,
-        plant_design_scenario=9,
+        plant_design_scenario=7,
         output_level=5,
     )
 
+    lcoe, lcoh, _, hi = run_simulation(config)
+
     # prior to 20240207 value was approx(10.823798551850347)
     # TODO base this test value on something. Currently just based on output at writing.
     with subtests.test("lcoh"):
-        assert lcoh == approx(12.583155204831298)
+        assert lcoh == approx(8.950730398944925)
 
     # prior to 20240207 value was approx(0.11035426429749774)
     # TODO base this test value on something. Currently just based on output at writing.
     with subtests.test("lcoe"):
-        assert lcoe == approx(0.1284376127848134)
-
-    with subtests.test("energy sources"):
-        expected_annual_energy_hybrid = (
-            hi.system.annual_energies.wind
-            + hi.system.annual_energies.wave
-            + hi.system.annual_energies.pv
-        )
-        assert hi.system.annual_energies.hybrid == approx(expected_annual_energy_hybrid)
+        assert lcoe == approx(0.12868090262684384)
 
 
 def test_simulation_wind_wave_solar_battery(subtests):
     filename_hopp_config_wind_wave_solar_battery = os.path.join(
         orbit_library_path, f"plant/hopp_config_wind_wave_solar_battery.yaml"
     )
 
-    lcoe, lcoh, _, hi = run_simulation(
-        filename_hopp_config_wind_wave_solar_battery,
-        filename_greenheart_config,
-        filename_turbine_config,
-        filename_orbit_config,
-        filename_floris_config,
+    config = GreenHeartSimulationConfig(
+        filename_hopp_config=filename_hopp_config_wind_wave_solar_battery,
+        filename_greenheart_config=filename_greenheart_config,
+        filename_turbine_config=filename_turbine_config,
+        filename_orbit_config=filename_orbit_config,
+        filename_floris_config=filename_floris_config,
         verbose=False,
         show_plots=False,
         save_plots=False,
         use_profast=True,
-        post_processing=False,
+        post_processing=True,
         incentive_option=1,
-        plant_design_scenario=10,
+        plant_design_scenario=7,
```

Comment:
why the design scenario change?

Reply:
I've addressed this and set back to 10. The test value has also gone back to close to original value (16.98)